src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/DumpTests.cs

```diff
@@ -74,5 +79,75 @@ public Task DumpTest(DiagnosticPortConnectionMode mode, DumpType type)
                     }
                 });
         }
+
+        [ConditionalTheory(typeof(TestConditions), nameof(TestConditions.IsDumpSupported))]
+        [InlineData(DiagnosticPortConnectionMode.Connect, DumpType.Mini)]
+        public Task DiagSessionTest(DiagnosticPortConnectionMode mode, DumpType type)
```

Comment:
Given that we're only testing a single mode and a single type of dump, should this test just be a Fact? Or do you intend to open this up later with additional modes/types?

Reply:
I am thinking that I would like to do this with Full dumps as well, since testing large artifacts may be worthwhile.